docker-compose.yml

```diff
@@ -647,7 +647,7 @@ services:
       CONTROL_CENTER_REST_SSL_KEYSTORE_LOCATION: /etc/kafka/secrets/kafka.controlcenter.keystore.jks
       CONTROL_CENTER_REST_SSL_KEYSTORE_PASSWORD: confluent
       CONTROL_CENTER_REST_SSL_KEY_PASSWORD: confluent
-      CONTROL_CENTER_SSL_CIPHER_SUITES: "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256"
+      CONTROL_CENTER_SSL_CIPHER_SUITES: "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
```

Comment:
This caught my attention, and - unrelated to this PR - I noticed that this has no effect, the variable name is not correct.  I've raised an issue and will fix this and assign the correct variable name to the new value.

Noticed because https://localhost:9022 worked fine on Chrome, but the other services did not, so Control Center was not using the unsupported cipher.